Calculate/Calculator.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Numerics;
+
+namespace Calculate;
+public class Calculator<T> where T : INumber<T>
+{
+    public IReadOnlyDictionary<char, Func<T, T, T>> MathematicalOperations { get; } = new Dictionary<char, Func<T, T, T>>
+    {
+        ['+'] = Add,
+        ['-'] = Subtract,
+        ['*'] = Multiply,
+        ['/'] = Divide
+    };
+
+#pragma warning disable CA1000
+    public static T Add(T a, T b) => a + b;
+    public static T Subtract(T a, T b) => a - b;
+    public static T Multiply(T a, T b) => a * b;
+    public static T Divide(T a, T b) => !b.Equals(0) ? a / b : throw new ArgumentException("Can't divide by 0", nameof(b));
+#pragma warning restore CA1000
+
+    public bool TryCalculate(string calculation, out float result)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(calculation, nameof(calculation));
+        string[] parts = calculation.Split(' ');
+        result = 0;
+        if(parts.Length != 3)
+        {
+           return false;
+        }
+        float lOperand;
+        float rOperand;
+        char operatorChar = parts[1][0];
+
+        if(!float.TryParse(parts[0], out lOperand) || !float.TryParse(parts[2], out rOperand))
```

Comment:
```suggestion
        if(float.TryParse(parts[0], out lOperand) && float.TryParse(parts[2], out rOperand))
```
I think this true and true is better than not true or not true